.github/workflows/publish_dockerhub_k8s_cache_main.yml

```diff
@@ -0,0 +1,58 @@
+# TODO: trigger only if testing or any other checks previously succeeded
+name: Push k8s cache to DockerHub (main)
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  id-token: write
+  packages: write
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: ebpf-instrument-k8s-cache
```

Comment:
According to this: https://github.com/open-telemetry/community/issues/2806#issuecomment-2968184963
I guess the image name should be changed to `opentelemetry-ebpf-k8s-cache`.